fixity/fixity.py

```diff
@@ -280,6 +279,25 @@ def main():
 
     session = Session()
 
+    logger = logging.getLogger(__name__)
+
+    # create console handler which logs stderr messages
```

Comment:
What's a `stderr message` in this context?

Reply:
warnings

Reply:
OK. I'd propose to either clarify these new comments better or remove them. Personally, I don't think they provide additional value if you consider the lines they're associated with versus the burden of maintaining comments.